assets/css/screen.css

```diff
@@ -152,90 +153,198 @@ body {
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    padding: 10vw 4vw;
+    z-index: 100;
+    padding: 6vw 3vw;
     min-height: 200px;
-    max-height: 450px;
-    text-align: center;
+    max-height: 340px;
 }
 
 .site-title {
     z-index: 10;
-    margin: 0;
+    margin: 0 0 0 -2px;
     padding: 0;
-    font-size: 3.8rem;
-    font-weight: 700;
+    font-size: 5.0rem;
+    font-weight: 600;
+    line-height: 1em;
 }
 
 .site-logo {
-    max-height: 45px;
+    max-height: 55px;
 }
 
 .site-description {
     z-index: 10;
     margin: 0;
     padding: 5px 0;
+    font-size: 2.1rem;
+    font-weight: 400;
+    line-height: 1.4em;
+    opacity: 0.8;
+}
+
+/* 4.1 Home header
+/* ---------------------------------------------------------- */
+
+.site-home-header {
+    z-index: 1000;
+}
+
+.site-home-header .site-header-background {
+    margin-top: 0;
+}
+
+.site-home-header .site-header-content {
+    padding: 5vw 3vw 6vw;
+}
+
+.site-home-header .site-title {
+    font-size: 5.5rem;
+    text-align: center;
+}
+
+.site-home-header .site-description {
     font-size: 2.2rem;
     font-weight: 300;
-    letter-spacing: 0.5px;
-    opacity: 0.8;
+}
+
+
+/* 4.2 Archive header (tag and author post lists)
+/* ---------------------------------------------------------- */
+
+.site-archive-header .site-header-content {
```

Comment:
to be checked: `.site-archive-header` refers to tag and author post lists, `.site-home-header` to home (design and behaviors are different). unsure about naming and if we have to differentiate at all, i.e. we could just have `.site-header` for archives and differentiate `.site-home-header` or vice versa

Reply:
I think this is ok